### PR TITLE
troubleshooting 

### PR DESCRIPTION
**Pull request by pwright**

### CODE REVIEW

File: modules/troubleshooting/pages/index.adoc [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+$ skupper status --namespace west
+
+Skupper is enabled for namespace "west" in interior mode. It is connected to 2 other sites. It has 1 exposed services.
+----
+
+The output shows:
+
+* A site exists in the specified namespace.
+* A link exists to two other sites.
+* A service is exposed on the {service-network}, and is accessible from this namespace.
```
Comment (briandooley) [inline modules/troubleshooting/pages/index.adoc:39]:
```suggestion
* A service is exposed on the {service-network} and is accessible from this namespace.
```
Passive voice also.

File: modules/troubleshooting/pages/index.adoc [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+
+.Procedure
+
+. Check the site status:
++
+--
+----
+$ skupper status --namespace west
+
+Skupper is enabled for namespace "west" in interior mode. It is connected to 2 other sites. It has 1 exposed services.
```
Comment (briandooley) [inline modules/troubleshooting/pages/index.adoc:32]:
```suggestion
Skupper is enabled for namespace "west" in interior mode. It is connected to 2 other sites. It has 1 exposed service.
```
Gotta be consistent on on using numerals or words to depict numbers. Numerals are used in this sentence but "two" is used a few sentences later. See https://www.ibm.com/docs/en/ibm-style?topic=measurement-numerals-versus-words

File: modules/troubleshooting/pages/index.adoc [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+. Check the link status:
++
+--
+----
+$ skupper link status --namespace east
+
+Link link1 is active
+----
+
+A link exists from the specified site to another site, meaning a token from another site was applied to the specified site.
```
Comment (briandooley) [inline modules/troubleshooting/pages/index.adoc:52]:
```suggestion
A link exists from the specified site to another site, meaning a token from another site applies to the specified site.
```
Passive voice.

File: modules/troubleshooting/pages/index.adoc [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+
+NOTE: Running `skupper link status` on a connected site only produces output if a token was used to create a link.
```
Comment (briandooley) [inline modules/troubleshooting/pages/index.adoc:54]:
```suggestion
NOTE: Running `skupper link status` on a connected site produces output only if a token was used to create a link.   
```
https://www.ibm.com/docs/en/ibm-style?topic=grammar-adverbs-only
Also, passive voice.

File: modules/troubleshooting/pages/index.adoc [hunk 5]
```diff
@@ -0,0 +1,170 @@
...
+         ╰─ Targets:
+            ├─ name: backend-77f8f45fc8-smckp
+            ├─ name: backend-77f8f45fc8-gh6tp
+            ╰─ name: backend-77f8f45fc8-m58tg
+----
+
+The output shows:
+
+* There are two sites on the {service-network}, `east` and `west`.
+* Details for each site, for example the namespace names.
```
Comment (briandooley) [inline modules/troubleshooting/pages/index.adoc:93]:
```suggestion
* Details for each site. For example, the namespace names.
```

File: modules/troubleshooting/pages/index.adoc [hunk 6]
```diff
@@ -0,0 +1,170 @@
...
+* The original services that are exposed (Targets), in this case the three backend services exposed using the `tcp` protocol.
```
Comment (briandooley) [inline modules/troubleshooting/pages/index.adoc:94]:
```suggestion
* The original services that are exposed (Targets). In this case, the three backend services exposed using the `tcp` protocol.
```

File: modules/troubleshooting/pages/index.adoc [hunk 7]
```diff
@@ -0,0 +1,170 @@
...
+* The services available on the {service-network}, including port, for example, `backend:8080`.
```
Comment (briandooley) [inline modules/troubleshooting/pages/index.adoc:95]:
```suggestion
* The services available on the {service-network}, including the port. For example, `backend:8080`. 
```

File: modules/troubleshooting/pages/index.adoc [hunk 8]
```diff
@@ -0,0 +1,170 @@
...
+* The original services that are exposed (Targets), in this case the three backend services exposed using the `tcp` protocol.
+* The services available on the {service-network}, including port, for example, `backend:8080`. 
+--
+
+
+// Type: procedure
+[id='creating-debug']
+== Creating a skupper debug tarball
+
+The debug tarball contains all the logs from the {skupper-name} components, and provides detailed information to help debug issues.
```
Comment (briandooley) [inline modules/troubleshooting/pages/index.adoc:103]:
```suggestion
The debug tarball contains all the logs from the {skupper-name} components and provides detailed information to help debug issues.
```

File: modules/troubleshooting/pages/index.adoc [hunk 9]
```diff
@@ -0,0 +1,170 @@
...
+The following issues and workarounds might help you debug simple scenarios when evaluating {skupper-name}.
+
+*Cannot initialize skupper*
+
+If the `skupper init` command fails, consider the following options:
+
+* Check the load balancer.
++
+--
+If you are evaluating {skupper-name} on minikube, a load balancer is not automatically created.
```
Comment (briandooley) [inline modules/troubleshooting/pages/index.adoc:132]: Passive voice.

File: modules/troubleshooting/pages/index.adoc [hunk 10]
```diff
@@ -0,0 +1,170 @@
...
+Use the following command to create a loadbalancer:
```
Comment (briandooley) [inline modules/troubleshooting/pages/index.adoc:133]:
```suggestion
Use the following command to create a load balancer:
```

File: modules/troubleshooting/pages/index.adoc [hunk 11]
```diff
@@ -0,0 +1,170 @@
...
+
+For other Kubernetes flavors, see the documentation from your provider.
+
+--
+
+* Initialize without ingress.
++
+--
+This option will prevent other sites from linking to this site, but linking outwards is supported.
+Once a link is established, traffic can flow in either direction.
```
Comment (briandooley) [inline modules/troubleshooting/pages/index.adoc:147]: Passive voice.

**Review by briandooley — CHANGES_REQUESTED** [review]

File: modules/troubleshooting/pages/index.adoc (hunk 1 above)
Comment (pwright) [reply modules/troubleshooting/pages/index.adoc:39]: dunno how to unpassive this one :(

File: modules/troubleshooting/pages/index.adoc (hunk 2 above)
Comment (pwright) [reply modules/troubleshooting/pages/index.adoc:32]: changed later instance 'two'->'2'
Cannot change this text as it's cli output

File: modules/troubleshooting/pages/index.adoc (hunk 3 above)
Comment (pwright) [reply modules/troubleshooting/pages/index.adoc:52]: doesn't work, need past tense, and in this case we don't know who did it... so leaving as is

File: modules/troubleshooting/pages/index.adoc (hunk 11 above)
Comment (pwright) [reply modules/troubleshooting/pages/index.adoc:147]: the future tense in the prev sentence is prob more egregious, and preventable ;)

**Pull request merged**